src/app/organizations/organizations.component.ts

```diff
@@ -23,18 +37,35 @@ export class OrganizationsComponent implements OnInit, OnDestroy {
   faAdd = faAdd;
   faSitemap = faSitemap;
   faUserGroup = faUserGroup;
+  faDollarSign = faDollarSign;
   private showJoinDialogSubscription?: Subscription;
+  organizations$?: Observable<OrganizationConfig[]>;
+  canAddOrganizations$?: Observable<boolean>;
 
   constructor(
     private store: Store,
     private dialogService: DialogService,
     private router: Router,
     private activatedRoute: ActivatedRoute,
-    public organizationCollectionService: OrganizationCollectionService,
-    public permissionService: OrganizationPermissionService
+    public organizationService: OrganizationCollectionService,
+    private organizationMembersService: OrganizationMembersCollectionService
   ) {}
 
   ngOnInit(): void {
+    this.canAddOrganizations$ = this.organizationService.canAddOrganizations$;
+
+    this.organizations$ = this.organizationService.getAllMemoized().pipe(
+      take(1),
+      map((orgs) =>
+        orgs.map((org) => {
+          return {
+            organization: org,
+            canEdit$: this.organizationService.canEditOrganization(org),
+            canViewMembers$: this.organizationMembersService.canViewMembers(org.metadata.name),
+          } as OrganizationConfig;
```

Comment:
Use satisfies instead of as:
https://dev.to/ayc0/typescript-49-satisfies-operator-1e4i

Reply:
good catch!